src/backend/renderer.js

```diff
@@ -1554,6 +1560,31 @@ export function attach(
     isProfiling = false;
   }
 
+  let suspendedIds = new Set();
+
+  function shouldSuspendFiber(fiber) {
+    if (suspendedIds.size === 0) {
+      return false;
+    }
+    // TODO: maybe optimize this.
+    const id = getFiberID(getPrimaryFiber(((fiber: any): Fiber)));
+    return suspendedIds.has(id);
+  }
+
+  function toggleSuspense(id) {
+    if (typeof overrideProps !== 'function') {
+      return;
+    }
+    if (suspendedIds.has(id)) {
+      suspendedIds.delete(id);
+    } else {
+      suspendedIds.add(id);
+    }
+    // Force a re-render.
+    const fiber = idToFiberMap.get(id);
+    overrideProps(fiber, [], null);
```

Comment:
I'm curious: why not inject a method like e.g. `suspendFiber(true)` that could be used as our signal for support as well as our method for re-rendering?

Reply:
I want to preserve our ability to do filtering based on something other than just putting it in a Set.